backend/serializers/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .participant_serializer import ParticipantSchema
```

Comment:
I understand, that we need to do that because `HacknightSchema` is not registered in Marshmallow registry, but think that we should either move all serializers to `backend/serializers.py` (as there not much of them and don't see a reason to have them in a package instead of a module) or at least us `__all__` and import all schemas from siblings modules.

Reply:
There was a discussion on this topic, and the decision was to keep serializers separately. So I sticked to that. We have hacknights nested in participants schema, and participants nested in hacknights schema. It caused circular imports, and that why I imported both schemas in __init__.py - to avoid circular import error.